pypeit/core/arc.py

```diff
@@ -387,7 +388,7 @@ def _plot(x, mph, mpd, threshold, edge, valley, ax, ind):
     # plt.grid()
     plt.show()
 
-
+# ToDO JFH nfitpix should be chosen based on the spectral sampling of the spectroscopic setup
 def detect_lines(censpec, nfitpix=5, sigdetect = 10.0, FWHM = 10.0, cont_samp = 30, nonlinear_counts=1e10, debug=False):
     """
     Extract an arc down the center of the chip and identify
```

Comment:
Is the doc string up-to-date, especially the Return's ?

Reply:
Docs added.